### PR TITLE
Docs: Remove Year in Review banner

### PR DESCRIPTION
**Pull request by ponori**
Given it's nearly March, it seems appropriate to remove the 2022 year in review banner.

### CODE REVIEW

File: docs/docs-components/AppLayout.js [hunk 1]
```diff
@@ -51,7 +50,7 @@ export default function AppLayout({ children, colorScheme }: Props): Node {
     <Box minHeight="100vh" color="default">
       <SkipToContent />
       <Header />
-      {isHomePage && <YearInReviewBanner />}
+      {isHomePage}
```
Comment (dangerismycat) [inline docs/docs-components/AppLayout.js:53]: This line needs to be removed completely

**Comment by ponori** [comment]
Thanks! Done.

**Comment by AlbertCarreras** [comment]
@ponori @dangerismycat Should we do deeper cleanup? Maybe not reverting the full PR.

There are still a few lines that should be cleanup for example https://github.com/pinterest/gestalt/pull/2558/files#diff-90409f25a385b55363f71806274620855df572340cc7f936fdb08c583445e58dR18

https://github.com/pinterest/gestalt/pull/2558/files#diff-ca731702917dd5c437bb5549907d0dae2a2d512305a5531bc2e34b7a9354dcc7

**Review by dangerismycat — APPROVED** [review]
@AlbertCarreras I think the rest of those changes from Ayesha's PR we would keep, as we want the Year in Review page to still be available — we just don't need to link to it from the home page. Perhaps we could add a link to it on the Roadmap page or something, though that could happen in a later PR.

**Pull request merged**